cmd: explicitly mention update is forbidden for repo imports (#2237)
iterative/dvc.org: content/docs/command-reference/update.md

  [transfer](/doc/command-reference/import-url#example-import-straight-to-the-remote)
  the new data directly to remote storage (the default one unless `-r` is used).
  No changes are done in the <abbr>workspace</abbr>. Use `dvc pull` to get the

>   data locally. This option can't be used with DVC or Git repository imports.

Reviewer: Thanks @isidentical I copy edited it a little.

Can we also start the paragraph with "update a `.dvc` file created with `dvc import-url` and" instead of the current opening?

Thanks

  [transfer](/doc/command-reference/import-url#example-import-straight-to-the-remote)
  the new data directly to remote storage (the default one unless `-r` is used).
  No changes are done in the <abbr>workspace</abbr>. Use `dvc pull` to get the

>   data locally. This option can't used when updating the data that is imported
>   from other DVC or Git repositories.

Reviewer: ```suggestion
  data locally. This option can't be used with DVC or Git repository imports.
```
